doc/WebpayPlus.md

```diff
@@ -0,0 +1,228 @@
+# Documentación adicional Webpay Plus
+
+Además de [la configuración usando los certificados de pruebas y el flujo de Webpay Plus Normal](../README.md#webpay-plus), el SDK permite también:
+
+- [Crear una transacción Webpay Plus Mall](#crear-una-transacción-webpay-plus-mall).
+- [Realizar transacciones con captura diferida](#realizar-transacciones-con-captura-diferida)
+- [Anular una transacción Webpay Plus](#anular-una-transaccion-webpay-plus)
+- [Configurar el ambiente Webpay de la integración](#configurar-el-ambiente-webpay-de-la-integración)
+  - [Credenciales del comercio](#credenciales-del-comercio).
+  - [Apuntar a producción](#apuntar-a-producción).
+
+
+## Crear una transacción Webpay Plus Mall
+
+En Webpay Plus Mall, puedes realizar cargos atribuibles a múltiples comercios dentro de una agrupación denominada *mall*. En la `Configuration` de `Webpay` se indica el código de comercio del *mall*. Y al realizar transacciones se asocian al código de comercio de cada *store* que pertenecen al *mall*.
+
+Para comenzar, obtener una instancia de `WebpayMallNormal` es muy similar a lo visto anteriormente para `WebpayNormal`:
+
+```java
+import cl.transbank.webpay.configuration.Configuration;
+import cl.transbank.webpay.Webpay;
+import cl.transbank.webpay.WebpayMallNormal;
+// ...
+WebpayMallNormal transaction =
+    new Webpay(Configuration.forTestingWebpayPlusMall()).getMallNormalTransaction();
+```
+
+Luego se deben crear las transacciones de los *stores*:
+
+```java
+import com.transbank.webpay.wswebpay.service.WsTransactionDetail;
+import com.transbank.webpay.wswebpay.service.WsInitTransactionOutput;
+
+
+// Configuration.forTestingWebpayPlusMall() configura un mall cuyos stores
+// tienen el código 597020000543 y 597020000544
+List<WsTransactionDetail> storeDetails = new ArrayList<WsTransactionDetail>();
+
+WsTransactionDetail detail = new WsTransactionDetail();
+detail.setCommerceCode("597020000543");
+detail.setAmount(2000);
+detail.setBuyOrder("identificador único de la orden para este store");
```

Comment:
It is not so clear where should I obtain it or how could I create it. And Why buyOrder here is an String while in the WebPayOneClick it is an integer or long value?

Reply:
The String vs Long is ugly but it's how it is not only in the SDK but in the SOAP webservices. Whoever did the oneclick stuff didn't bother to keep consistency.

I'll add comments to make it clear that the buy order is generated by the merchant. Note that every sale in Chile usually has an order number so integrators shouldn't have problems getting the number (albeit with the "mall" mode is a bit trickier)